meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/imx8mmevk/bootloader.uuu.in

```diff
@@ -1,23 +1,20 @@
 uuu_version 1.2.39
 
-SDP: boot -f imx-boot-mfgtool
+SDP: boot -f SPL-mfgtool
 
-# These commands will be run when use SPL and will be skipped if no spl
-# SDPU will be deprecated. please use SDPV instead of SDPU
-SDPU: delay 1000
-SDPU: write -f imx-boot-mfgtool -offset 0x57c00
-SDPU: jump
-
-# These commands will be run when use SPL and will be skipped if no spl
-# if (SPL support SDPV)
 SDPV: delay 1000
-SDPV: write -f imx-boot-mfgtool -skipspl
+SDPV: write -f u-boot-mfgtool.itb
 SDPV: jump
 
 FB: ucmd setenv fastboot_dev mmc
 FB: ucmd setenv mmcdev ${emmc_dev}
 FB: ucmd mmc dev ${emmc_dev}
-FB: flash bootloader ../imx-boot-@@MACHINE@@
+FB: ucmd setenv set_blkcnt 'setexpr blkcnt 0x${filesize} + 0x1ff && setexpr blkcnt ${blkcnt} / 0x200'
```

Comment:
This logic can be replaced with 2 commands:
```
FB: flash bootloader ../SPL-@@MACHINE@@
FB: flash bootloader2 ../u-boot-@@MACHINE@@
```
We need to set this in the standard lmp.cfg:
```
CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x300
```
And set this in the mfgtool lmp.cfg:
```
CONFIG_FSL_FASTBOOT_BOOTLOADER2=y
CONFIG_FSL_FASTBOOT_BOOTLOADER2_OFFSET=0x300
```

Reply:
Now I got it how it works, looks like bootloader/bootloader2 is not boot0/boot1 (what I assumed initially), but just some defined for the offsets, nice.

But how did that work before, when we were using boot0 for imx-boot?